<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>SpaceSkyNet</author>
 <class>setupMainWindow</class>
 <widget class="QMainWindow" name="setupMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="languageFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="languageComboBox">
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese(Simplified)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="songsDBModeFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="songsDBModeLabel">
        <property name="text">
         <string>Songs DB Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="songsDBModeComboBox">
        <item>
         <property name="text">
          <string notr="true">osu!</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">danser</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QFormLayout" name="osuRootPathFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="osuRootPathLabel">
        <property name="text">
         <string>osu! Root Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QClickedLineEdit" name="osuRootPathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QFormLayout" name="skinsPathFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="skinsPathLabel">
        <property name="text">
         <string>Skins Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QClickedLineEdit" name="skinsPathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QFormLayout" name="songsPathFormLayout">
      <item row="0" column="1">
       <widget class="QClickedLineEdit" name="songsPathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="songsPathLabel">
        <property name="text">
         <string>Songs Path:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QFormLayout" name="danserPathFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="danserPathLabel">
        <property name="text">
         <string>danser Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QClickedLineEdit" name="danserPathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>GuiWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
